### PR TITLE
Dev branch judd

### PR DESCRIPTION
**Pull request by judd-gnizama**
Corrected spelling

### CODE REVIEW

**Review by famasboy888 — COMMENTED** [review]
No i dont like. You do

File: README.md [hunk 1]
```diff
@@ -23,4 +23,4 @@
 - git remote add origin https://github.com/judd-gnizama/mern_court_control.git
 - git push -u origin main
 
-//END
+//AND I will always love youuu..
```
Comment (famasboy888) [inline README.md:26]: No i dont like

**Comment by judd-gnizama** [comment]
Whuutt

**Review by famasboy888 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (judd-gnizama) [reply README.md:26]: Replied here

Comment (judd-gnizama) [reply README.md:26]:
```suggestion
//This is a suggestion
```

**Pull request merged**